src/api/helpers/create-a-different-trading-address/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Context } from '../../types';
+
+/**
+ * createADifferentTradingAddress
+ * Create a different trading address with company relationships.
+ * @param {Object} KeystoneJS context API
+ * @param {Object} Company address data
+ * @param {String} Company ID
+ * @returns {Object} Created different trading address
```

Comment:
Only 2 params passed to this function

Reply:
Can we align this naming and the file name to be `CompanyDifferentTradingAddress` since that's what it's creating?

So e.g instance of `createADifferentTradingAddress` are `createACompanyDifferentTradingAddress`